restaurants/src/app/services/restaurants.service.ts

```diff
@@ -1,15 +1,45 @@
 import { Restaurant } from "../models/restaurants.model";
 import { Component } from "@angular/core";
+import { v4 as uuidv4 } from 'uuid';
+import { Injectable } from "@angular/core";
+import { __values } from "tslib";
 
-export const RESTAURANTS: Restaurant[] = [
-{ id: 1, name: 'Broders', rank: 7.9},
-{ id: 2, name: 'Taqueria y birreria las cuatro', rank: 8.3},
-{ id: 3, name: 'The Luscious Crab', rank: 0.1},
-{ id: 4, name: "Famous Dave's", rank: 0.1},
-{ id: 5, name: 'Fogo de Chao', rank: 0.1},
-{ id: 6, name: 'Demera Ethopian Restaurant', rank: 0.1},
-{ id: 7, name: 'WingStop', rank: 8.9},
-{ id: 8, name: 'MT Noodles', rank: 0.1},
-{ id: 9, name: 'Hmong Village',rank: 0.1},
-{ id: 10, name: 'Gyu-kaku Japanese BBQ', rank: 9.0}
-];
+// TODO: add @Injectable...
+
+@Injectable({
+    providedIn: 'root',
+}
+
```

Comment:
Extra empty line can be removed - end parentheses can be on the same line as the curly brace (line 11)

Reply:
yes!